prom_export/info.json

```diff
@@ -4,12 +4,12 @@
     ],
     "description": "Exposes a HTTP endpoint for exporting guild metrics in Prometheus format",
     "short": "Exports guild metrics in Prometheus format",
-    "name": "Prom Exporter",
-
+    "name": "prom_export",

```

Comment:
technically `_exporter` would be more correct, in alignment with https://github.com/prometheus-community?q=exporter&type=all&language=&sort=